pipeline-scripts/buildlib.groovy

```diff
@@ -724,6 +724,27 @@ def build_ami(major, minor, version, release, yum_base_url, ansible_branch, mail
     }
 }
 
+def sweep(build_version, sweep_builds) {
+    def sweep_job = build(
+        job: 'build%2Fsweep',
+        prapagate: false,
+        parameters: [
+            param('String', 'BUILD_VERSION', build_version),
+            param('Boolean', 'SWEEP_BUILDS', sweep_builds),
+        ]
+    )
+    if (sweep_job.result != 'SUCCESS') {
+        commonlib.email(
+            replyTo: mail_list,
+            to: 'aos-art-automation+failed-sweep@redhat.com',
+            from: 'aos-art-automation@redhat.com',
+            subject: "Problem sweeping after ${currentBuild.displayName}",
+            body: "Jenkins console: ${commonlib.buildURL('console')}",
+        )
+        currentBuild.result = 'UNSTABLE'
```

Comment:
IMO sending emails should honor the suppress email parameter.

Reply:
It will automatically. Check out `commonlib.groovy` where `def email(args) {` is defined to see how (It looks at `params.SUPPRESS_EMAIL`)

Reply:
OK.